offchain/rollups-events/tests/certs/gen-test-certs.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
```

Comment:
Let's add this file in a separate commit. And if you modified the original file, add the modifications in the `feat(rollups-events): connect to Redis via TLS` commit.

Reply:
As a suggestion, I'd also set the author of the commit to be the person/project who wrote this component to make sure we give credit to them as well.